ddlmaker.go

```diff
@@ -25,22 +25,22 @@ type DDLMaker struct {
 	Tables  []dialect.Table
 }
 
-// NewMaker return DDLMaker
-func NewMaker(conf Config) (*DDLMaker, error) {
-	dialect, err := dialect.NewDialect(conf.DB.Driver, conf.DB.Engine, conf.DB.Charset)
+// New creates a DDLMaker and returns it.
+func New(conf Config) (*DDLMaker, error) {
+	d, err := dialect.New(conf.DB.Driver, conf.DB.Engine, conf.DB.Charset)
```

Comment:
The variable name of `dialect` is equal to the package name of `dialect` and shadows the declaration of the package. So, this variable is renamed to `d`.